packages/css-library/src/tokens/color.scss

```diff
@@ -1,27 +1,27 @@
-$color-orange: #eb7f29;
+@use "./_variables" as *;
 
 // Our colors
 $tokens-color-basic: (
-  "base": "#212121",
-  "white": "#ffffff",
-  "black": "#000000",
+  "base": $color-base,
+  "white": $color-white,
+  "black": $color-black,
   "orange": false,
```

Comment:
This is just one possible way to use the tokens. We could also have style dictionary build a [SASS map format](https://amzn.github.io/style-dictionary/#/formats?id=scssmap-flat), but that would put all tokens (colors, spacing, etc.) into the same map which we may not want. A custom formatter may allow us to have multiple SASS maps build in the same file.

If we went in that direction we may be able to have our custom formatter recognize [custom token attributes](https://amzn.github.io/style-dictionary/#/tokens?id=design-token-attributes) to allow us to do something like:

```json
"orange": { "value": "#eb7f29", "attributes": { "utilityClass": false } }
```